index.js

```diff
@@ -80,7 +80,10 @@ Electron.prototype._spawn = function(url){
   debug('spawn %s', url);
 
   var self = this;
-  var ps = self.ps = spawn(electron, [runner], {
+  var electronOpts = []
```

Comment:
I think it's simpler if we build an args array instead:

```js
const args = [runner]
```

And then push to this